src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -150,6 +158,11 @@ public bool IsADPAnalyticsReportingApproved
         /// </summary>
         public bool ShowPreviewBubbles { get; set; }
 
+        /// <summary>
+        /// Indicates if Revit units should be used for graphic helpers for Dynamo Revit
+        /// </summary>
+        public bool UseRevitScaleUnits { get; set; }
```

Comment:
I think it's strange that Revit should be referenced directly in Dynamo core?

Reply:
Hey @mjkkirschner - I am afraid this whole process requires for some form of awareness on the Dynamo side that this is a Revit session. Basically, the task is to make the grid layout behave differently under Revit, that's how I understand it. Maybe this core concept is no good, and then we may have to go back a bit and analyze the approach once again? @Amoursol

Reply:
@mjkkirschner agree that having a Revit reference in Core isn't ideal. Is there a way to inject this functionality in from D4R instead?

Reply:
Hi @mjkkirschner - I replaced 'Revit' to 'Host' for a platform-agnostic approach, I hope that addresses the concern around semantics, and sorry if I didn't understand correctly where the issue was coming from. I hope I got it right this time around!